controller/src/main/java/io/pravega/controller/store/stream/PravegaTablesStream.java

```diff
@@ -131,20 +131,20 @@
     private final ScheduledExecutorService executor;
 
     @VisibleForTesting
-    PravegaTablesStream(final String scopeName, final String streamName, PravegaTablesStoreHelper storeHelper, 
+    PravegaTablesStream(final String scopeName, final String streamName, PravegaTablesStoreHelper storeHelper,
```

Comment:
there doesnt seem to be any change in this class. please revert it.
also if possible, please revert the formatting changes in other classes, particularly PersistentStreamBase and PravegaTablesScope.

Reply:
done.